src/main/java/com/example/Library/Management/System/controller/LibraryController.java

```diff
@@ -18,12 +18,12 @@ public class LibraryController {
     private final LibraryService libraryService;
 
     //Get
-    @GetMapping("/viewAllLibraries")
+    @GetMapping("/view-all")
     public List<LibraryRequest> getMapping() {
         return libraryService.viewAllLibraries();
     }
 
-    @GetMapping("/viewLibraryById/{id}")
+    @GetMapping("/view-by-id/{id}")
     public LibraryRequest getMappingById(@PathVariable Long id){
```

Comment:
applies to all methods, kindly rename to represent it's usage 
```suggestion
    public LibraryDTO getLibraryById(@PathVariable Long id){
```